I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 159, in script
    jump after
ScriptError: could not find label 'after'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 159, in script
    jump after
  File "D:\Ren'Py\renpy-8.3.7-sdk\renpy\ast.py", line 1712, in execute
    rv = renpy.game.script.lookup(target)
  File "D:\Ren'Py\renpy-8.3.7-sdk\renpy\script.py", line 1103, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'after'.

Windows-10-10.0.19045 AMD64
Ren'Py 8.3.7.25031702
Bug Wife Turbo Space Hell 0.2
Thu Jun 26 22:42:56 2025
